### PR TITLE
feat(presenter): sixth section with content

### LINKED ISSUE #32: 🚀 [Feature] Section6
## 🚀 Feature

image

### PR DESCRIPTION
**Pull request by maeckes-infinitylabs**
## 🍰 Pullrequest

Added sixth section with content

image

### Issues

- fixes #32 

### Todo

- [X] None

### CODE REVIEW

File: presenter/src/components/sections/FifthSection.vue [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<template>
+  <div class="">
+    <div class="section5 h-screen pt-4">{{ $t('home.section4.headline') }}</div>
```
Comment (ulfgebhardt) [inline presenter/src/components/sections/FifthSection.vue:3]:
```suggestion
    <div class="section5 h-screen pt-4">{{ $t('home.section5.headline') }}</div>
```

Is this on purpose? Or just a typo?

File: presenter/src/pages/index/index.page.vue [hunk 2]
```diff
@@ -12,6 +12,9 @@
     <div id="contact">
       <Section4 />
     </div>
```
Comment (ulfgebhardt) [inline presenter/src/pages/index/index.page.vue:14]: Where is section 5?

File: presenter/src/components/sections/FifthSection.vue (hunk 1 above)
Comment (maeckes-infinitylabs) [reply presenter/src/components/sections/FifthSection.vue:3]: I see that i pushed 5th section already. this was a mistake for this PR. shall i delete this from this pr?
to keep this here clean?

File: presenter/src/pages/index/index.page.vue (hunk 2 above)
Comment (maeckes-infinitylabs) [reply presenter/src/pages/index/index.page.vue:14]: this PR should only contain section 6 related components (see other comment).

**Review by ulfgebhardt — APPROVED** [review]

File: presenter/src/locales/en.json [hunk 3]
```diff
@@ -24,6 +24,15 @@
     },
     "section4": {
       "headline": "Contact Section Content"
+    },
+    "section5": {
+      "headline": "Kontakt"
+    },
```
Comment (ulfgebhardt) [inline presenter/src/locales/en.json:30]: here is still a section5 thing. Is this ment to be?

File: presenter/src/locales/de.json [hunk 4]
```diff
@@ -24,6 +24,15 @@
     },
     "section4": {
       "headline": "Kontakt"
+    },
+    "section5": {
+      "headline": "Kontakt"
+    },
```
Comment (ulfgebhardt) [inline presenter/src/locales/de.json:30]: section5 content

Comment (maeckes-infinitylabs) [reply presenter/src/locales/de.json]: thx...removed it

**Review by ulfgebhardt — APPROVED** [review]

**Pull request merged**